flink-table-store-hive/src/test/java/org/apache/flink/table/store/FileStoreTestHelper.java

```diff
@@ -48,37 +50,33 @@
 import java.util.concurrent.Executors;
 import java.util.function.BiFunction;
 import java.util.function.Function;
-import java.util.stream.Collectors;
 
 /** Helper class to write and read {@link RowData} with {@link FileStoreImpl}. */
 public class FileStoreTestHelper {
 
-    private final FileStoreImpl store;
+    private final FileStoreTable table;
+    private final FileStore store;
     private final BiFunction<RowData, RowData, BinaryRowData> partitionCalculator;
     private final Function<RowData, Integer> bucketCalculator;
     private final Map<BinaryRowData, Map<Integer, RecordWriter>> writers;
     private final ExecutorService compactExecutor;
 
     public FileStoreTestHelper(
             Configuration conf,
-            RowType partitionType,
-            RowType keyType,
-            RowType valueType,
-            MergeFunction mergeFunction,
+            RowType rowType,
+            List<String> partitionKeys,
+            List<String> primaryKeys,
             BiFunction<RowData, RowData, BinaryRowData> partitionCalculator,
-            Function<RowData, Integer> bucketCalculator) {
-        FileStoreOptions options = new FileStoreOptions(conf);
-        this.store =
-                new FileStoreImpl(
-                        options.path().toString(),
-                        0,
-                        options,
-                        WriteMode.CHANGE_LOG,
-                        UUID.randomUUID().toString(),
-                        partitionType,
-                        keyType,
-                        valueType,
-                        mergeFunction);
+            Function<RowData, Integer> bucketCalculator)
+            throws Exception {
+        Path tablePath = FileStoreOptions.path(conf);
+        Schema schema =
```

Comment:
Can we have a `FileStoreTableFactory`?
We don't need to make `SchemaManager` public.